diceRoll/diceRollUnitTests.js

```diff
@@ -46,14 +153,15 @@ function UnitTest(testName, functionToBeTested) {
   this.functionToBeTested = functionToBeTested;
   this.report;
   try {
-    if (!functionToBeTested()) {
```

Comment:
I don't understand enough of what's going on with that code to fix what's wrong with it. Pulling this file as is for now. Maybe you can explain this to me later.